omod/src/main/java/org/openmrs/web/controller/maintenance/ServerLogController.java

```diff
@@ -76,6 +76,7 @@ protected ModelAndView onSubmit(HttpServletRequest request, HttpServletResponse
 				}
 				
 				MemoryAppender memoryAppender = (MemoryAppender) getMemoryAppenderHandle.invoke();
+				log.info("Server log was accessed");
```

Comment:
is there a better method of having this than simply hard coding stuff,ie under https://github.com/openmrs/openmrs-module-legacyui/blob/master/api/src/main/resources/messages.properties ,hi  @isears what do you think?

Reply:
@HerbertYiga that's a fair point, but it seems like most existing calls to `log.info()` use hard-coded strings rather than message properties ([examples](https://github.com/search?p=1&q=org%3Aopenmrs+log.info&type=Code)).